Doc/library/urllib.parse.rst

```diff
@@ -348,19 +349,22 @@ or on combining URL components into a URL string.
    with an empty query; the RFC states that these are equivalent).
 
 
-.. function:: urljoin(base, url, allow_fragments=True)
+.. function:: urljoin(base, url, allow_fragments=True, classes=SchemeFlag(0))
```

Comment:
Why is this called `classes` here and `flags` above?  Consistency is important.  These should also be made keyword only arguments.

Reply:
I forgot to update after renaming from classes to flags. I'll change it over to `behavior_overrides`.